<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibFile</class>
 <widget class="QDialog" name="CalibFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalibFile</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>4</y>
     <width>561</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Soil + Geology</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>34</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Porosity</string>
    </property>
    <property name="text">
     <string>Porosity</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>201</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Soil</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>30</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Vertical Soil Conductivity</string>
     </property>
     <property name="text">
      <string>KsatV</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>54</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Vertical macropore Conductivity</string>
     </property>
     <property name="text">
      <string>mpKsatV</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>116</x>
       <y>30</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Infiltration Depth</string>
     </property>
     <property name="text">
      <string>infl D</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>116</x>
       <y>54</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>horizontal macropore fraction</string>
     </property>
     <property name="text">
      <string>mpfH</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="soilKsatV">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>27</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="macKsatV">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>50</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="infD">
     <property name="geometry">
      <rect>
       <x>158</x>
       <y>27</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="hAreaf">
     <property name="geometry">
      <rect>
       <x>158</x>
       <y>50</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>34</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Van genuchten parameters</string>
    </property>
    <property name="text">
     <string>Alpha</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>409</x>
      <y>34</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Van genuchten parameters</string>
    </property>
    <property name="text">
     <string>Beta</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPorosity">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>31</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAlpha">
    <property name="geometry">
     <rect>
      <x>276</x>
      <y>31</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBeta">
    <property name="geometry">
     <rect>
      <x>456</x>
      <y>31</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>217</x>
      <y>60</y>
      <width>331</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Geology</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>30</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Horizontal Conductivity</string>
     </property>
     <property name="text">
      <string>KsatH</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>54</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Vertical Conductivity</string>
     </property>
     <property name="text">
      <string>KsatV</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>117</x>
       <y>30</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>macropore Conductivity</string>
     </property>
     <property name="text">
      <string>mpKsatH</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>117</x>
       <y>54</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Macropore Depth</string>
     </property>
     <property name="text">
      <string>mpDepth</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>237</x>
       <y>30</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>vertical macropore fraction</string>
     </property>
     <property name="text">
      <string>mpfV</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="geolKsatH">
     <property name="geometry">
      <rect>
       <x>58</x>
       <y>27</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="geolKsatV">
     <property name="geometry">
      <rect>
       <x>58</x>
       <y>50</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="macKsatH">
     <property name="geometry">
      <rect>
       <x>185</x>
       <y>27</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="macD">
     <property name="geometry">
      <rect>
       <x>185</x>
       <y>50</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="vAreaF">
     <property name="geometry">
      <rect>
       <x>288</x>
       <y>27</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>152</y>
     <width>561</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Land Cover</string>
   </property>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>30</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Root Zone Depth</string>
    </property>
    <property name="text">
     <string>RzDepth</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="RzD">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>28</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>147</x>
      <y>30</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Vegetation Fraction</string>
    </property>
    <property name="text">
     <string>VegFrac</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>147</x>
      <y>55</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Albedo</string>
    </property>
    <property name="text">
     <string>Albedo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Manning’s roughness (n)</string>
    </property>
    <property name="text">
     <string>Roughness</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>55</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Interception Loss</string>
    </property>
    <property name="text">
     <string>Evap Canopy</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>413</x>
      <y>30</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Transpiration</string>
    </property>
    <property name="text">
     <string>Transpiration</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>413</x>
      <y>55</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Evaporation from Ground</string>
    </property>
    <property name="text">
     <string>Evap Ground</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="VegFrac">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>28</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Albedo">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>53</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Rough">
    <property name="geometry">
     <rect>
      <x>367</x>
      <y>28</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Et0">
    <property name="geometry">
     <rect>
      <x>367</x>
      <y>53</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Et1">
    <property name="geometry">
     <rect>
      <x>507</x>
      <y>28</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Et2">
    <property name="geometry">
     <rect>
      <x>507</x>
      <y>53</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>232</y>
     <width>561</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>River</string>
   </property>
   <widget class="QLineEdit" name="rivRough">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>27</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rivBedThickness">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>52</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rivKsatH">
    <property name="geometry">
     <rect>
      <x>289</x>
      <y>27</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rivKsatV">
    <property name="geometry">
     <rect>
      <x>289</x>
      <y>52</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rivDepth">
    <property name="geometry">
     <rect>
      <x>507</x>
      <y>27</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rivWidCoeff">
    <property name="geometry">
     <rect>
      <x>507</x>
      <y>52</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>30</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>River bed Manning’s roughness (n)</string>
    </property>
    <property name="text">
     <string>Rough</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>54</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>River bed thickness</string>
    </property>
    <property name="text">
     <string>Thickness</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Conductivity of river sides</string>
    </property>
    <property name="text">
     <string>KsatH</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>54</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Conductivity of river bed</string>
    </property>
    <property name="text">
     <string>KsatV</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>427</x>
      <y>30</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>River depth</string>
    </property>
    <property name="text">
     <string>Depth</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>427</x>
      <y>54</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>River width</string>
    </property>
    <property name="text">
     <string>Width</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Output .CALIB File</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>561</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Forcing</string>
   </property>
   <widget class="QLineEdit" name="Precep">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>29</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Temp">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>29</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>31</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Precipitation Timeseries</string>
    </property>
    <property name="text">
     <string>Precipitation</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>31</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Temperature timeseries</string>
    </property>
    <property name="text">
     <string>Temperature</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="calibFile">
   <property name="geometry">
    <rect>
     <x>157</x>
     <y>386</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBrowse">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>382</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>414</y>
     <width>581</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonHelp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>438</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRun">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>438</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>438</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
